alarm/utils.py

```diff
@@ -33,3 +30,58 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def create_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in thresholds:
+        ThresholdBrake.objects.create(threshold=threshold)
+
+
+def get_thresholds_brake_prices(trade_pair):
+    thresholds_brake_prices = ThresholdBrake.objects.filter(threshold__trade_pair=trade_pair).order_by(
```

Comment:
It gives all threshold brakes, not the recent ones.

Reply:
Fixed